Presentation.Web/app/components/it-contract/it-contract-plan.controller.ts

```diff
@@ -521,24 +515,24 @@
                         filterable: false
                     },
                     {
-                        field: "AssociatedInterfaceUsages",
-                        title: "Antal udstillede snitflader",
-                        width: 50,
-                        persistId: "numOfItInterfacesExhibit", // DON'T YOU DARE RENAME!
+                        field: "AssociatedInterfaces",
+                        title: "Antal snitflader",
+                        width: 60,
+                        persistId: "numberOfInterfaces", // DON'T YOU DARE RENAME!
                         template: dataItem => {
-                            return dataItem.AssociatedInterfaceExposures.length.toString();
+                            return _.uniqBy(dataItem.AssociatedSystemRelations.filter(x => x.RelationInterfaceId != null), x => x.RelationInterfaceId).length.toString();
                         },
                         attributes: { "class": "text-center" },
                         sortable: false,
                         filterable: false
                     },
                     {
-                        field: "AssociatedInterfaceExposures",
-                        title: "Antal anvendte snitflader",
-                        width: 50,
-                        persistId: "numOfItInterfacesUsages", // DON'T YOU DARE RENAME!
+                        field: "AssociatedRelations",
+                        title: "Antal relationer",
+                        width: 60,
+                        persistId: "numberOfRelations", // DON'T YOU DARE RENAME!
                         template: dataItem => {
-                            return dataItem.AssociatedInterfaceUsages.length.toString();
+                            return dataItem.AssociatedSystemRelations.length.toString();
```

Comment:
Hvad hvis dataitem er null?

Reply:
Det er en del af kendo. Så koden her bliver kørt for hvert contract element der kommer ud af odata kaldet. Så hvis ikke der er et dataitem så bliver det ikke kørt